### ISSUE TITLE
/txns.json doesn't work

### ISSUE THREAD

**Issue by yegor256** [zold-io/zold#624]
After #623:

```
Unexpected error while processing request: undefined method `bytesize' for #<Hash:0x00007f3b8ad537d0>
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:166:in `block in finish'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:166:in `each'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:166:in `inject'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:166:in `finish'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:935:in `call!'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:913:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-2.0.6/lib/rack/deflater.rb:34:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/xss_header.rb:18:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/path_traversal.rb:16:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/json_csrf.rb:26:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/base.rb:50:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/base.rb:50:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-protection-2.0.4/lib/rack/protection/frame_options.rb:31:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-2.0.6/lib/rack/null_logger.rb:9:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/rack-2.0.6/lib/rack/head.rb:12:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:194:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:1957:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:1502:in `block in call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:1729:in `synchronize'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/sinatra-2.0.4/lib/sinatra/base.rb:1502:in `call'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/thin-1.7.2/lib/thin/connection.rb:86:in `block in pre_process'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/thin-1.7.2/lib/thin/connection.rb:84:in `catch'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/thin-1.7.2/lib/thin/connection.rb:84:in `pre_process'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/thin-1.7.2/lib/thin/connection.rb:50:in `block in process'
	/home/yegor/.rvm/gems/ruby-2.5.1/gems/eventmachine-1.2.7/lib/eventmachine.rb:1077:in `block in spawn_threadpool'
```

**Comment by yegor256** [comment]
@rultor release, tag is `0.18.4`

**Comment by 0crat** [comment]
Job was finished in 2 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#624](https://github.com/zold-io/zold/issues/624) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)
